.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,50 @@
+name: Bump Version Tag
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v6
+        with:
+          node-version: '24'
+
+      - name: Get latest version tag
+        id: get_version
+        run: |
+          git fetch --tags
+          LATEST_TAG=$(git tag -l "v*" | sort -V | tail -n 1)
+          if [ -z "$LATEST_TAG" ]; then
+            echo "latest=v0.0.0" >> $GITHUB_OUTPUT
+          else
+            echo "latest=$LATEST_TAG" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Bump patch version
+        id: bump_version
+        run: |
+          LATEST_VERSION="${{ steps.get_version.outputs.latest }}"
+          VERSION_NUMBER="${LATEST_VERSION#v}"
+
+          echo "$VERSION_NUMBER" > VERSION_TEMP
+          npm version --no-git-tag-version $VERSION_NUMBER
+          npm version --no-git-tag-version patch
```

Comment:
Should we make these `prerelease` versions with `--preid=dev` instead of patch versions? That would mean that by default, people installing VolView in their project would get a stable manual release rather than the cutting-edge main branch.